tests/trx_generator/trx_generator.cpp

```diff
@@ -97,8 +97,21 @@ namespace eosio::testing {
       _stop_on_trx_failed(stop_on_trx_failed) {
    }
 
+   void log_first_trx(const std::string& log_dir, const chain::signed_transaction& trx) {
+      std::ostringstream fileName;
+      fileName << log_dir << "/first_trx_" << getpid() << ".txt";
+      std::ofstream out(fileName.str());
+
+      out << fc::string(trx.id()) << "\n";
+      out.close();
+   }
+
    void transfer_trx_generator::push_transaction(p2p_trx_provider& provider, signed_transaction_w_signer& trx, uint64_t& nonce_prefix, uint64_t& nonce, const fc::microseconds& trx_expiration, const chain_id_type& chain_id, const block_id_type& last_irr_block_id) {
       update_resign_transaction(trx._trx, trx._signer, ++nonce_prefix, nonce, trx_expiration, chain_id, last_irr_block_id);
+      if (_txcount == 0)
+      {
```

Comment:
Odd, change of style for this `if`.

Reply:
Addressed: [de3a1c8](https://github.com/AntelopeIO/leap/pull/639/commits/de3a1c8392564d6d0ab41412374948c4c8f5359c)